lib/tlog/es_json_reader.c

```diff
@@ -230,6 +231,43 @@ tlog_es_json_reader_init(struct tlog_json_reader *reader, va_list ap)
         goto error;
     }
 
+    /* Allow kerberos (negotiate) authentication */
```

Comment:
Can you please verify this works with kerberos authentication on your side?

Reply:
Yes, I tested this with one of our kerberos enabled ES instances. I'm kind of surprised nobody has requested kerberos support and wonder how others control access to tlog indexes :)